Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml

```diff
@@ -86,43 +95,92 @@
         <!-- Spring framework -->
         <dependency>
             <groupId>org.springframework</groupId>
-            <artifactId>spring-core</artifactId>
```

Comment:
Our code doesn't directly use spring-core packages (org.springframework.core.*).  That jar is a transitive dependency to spring-beans, however, so the jar is still included in the lib folder in the final build.